Package.swift

```diff
@@ -5,7 +5,6 @@ import PackageDescription
 
 let package = Package(
     name: "NostrSDK",
-    platforms: [.macOS(.v12)],
```

Comment:
This came from pulling in SwiftLint initially, which got pulled out, but we forgot to remove this line.